base/pkg/dir.jl

```diff
@@ -70,9 +67,7 @@ end
 
 function getmetabranch()
     try
-        open(joinpath(path(),"META_BRANCH")) do io
-          chomp(readuntil(io, "/n"))
-        end
+        chomp(readline(joinpath(path(),"META_BRANCH")))
```

Comment:
...and good heavens, is that a `/n` instead of `\n` on the line above? Seems like it worked because the data it wants is actually the whole file (usually?). But wow.